hs_modelinstance/tests/test_modelinstance_metadata.py

```diff
@@ -49,12 +49,19 @@ def setUp(self):
         )
 
         self.temp_dir = tempfile.mkdtemp()
+
         self.file_name = "MIR.txt"
         temp_text_file = os.path.join(self.temp_dir, self.file_name)
         text_file = open(temp_text_file, 'w')
         text_file.write("Model Instance resource files")
         self.text_file_obj = open(temp_text_file, 'r')
 
+        self.file_name_2 = "MIR2.txt"

```

Comment:
Well, I'd rather not get into that. I'm trying to insure that we won't have a merge problem, not correct others' mistakes. Further, I would have to construct a valid file before I could ensure that future testing wouldn't break. Will add TODOs.